tasks/2017/spread-the-word.md

```diff
@@ -0,0 +1,64 @@
+# GCI Tasks: Spread The Word
+
+## Background
+
+Upvotes and reviews make a big difference in the public perception of projects
+like Zulip.
+
+## Task description
+
+In this task, we will ask you to rate and review Zulip on some technology sites.
+You're free to decide whether or not to do each individual task, but we do ask
+that you do at least five of the social media marketing tasks in order to get
+credit for the overall task. We expect completing all the tasks will take about
+90 minutes.
+
+### Star us on GitHub.
+
+There are four main repositories: [server/web](https://github.com/zulip/zulip),
+[mobile](https://github.com/zulip/zulip-mobile),
+[desktop](https://github.com/zulip/zulip-electron), and [Python
+API](https://github.com/zulip/python-zulip-api). For each repository, star the
+repository, and then write down the number of stars that repository has.
+
+### Share Zulip on StackShare
+
+1. Create an account at StackShare. Add your name and a picture, [i.e. do the
+work to have a real profile].
+
+2. Think of six technology products you really like, and consider adding Zulip
+to that list.
+
+3. For each product in 2: Add the product to your Stack, star it, and upvote at
+least three reasons why people like product that you find compelling.
+
+4. Send us a link with your profile (e.g. https://stackshare.io/Lyla-Fischer,
+but replacing Lyla-Fischer with your username)
+
+### Write a review on the android play store
+
+    Go to the Zulip app’s entry in the play store, scroll down to the “my
+    review” section, and take a screenshot of your review.
+
+### Write a review on the ios app store.
+
+    Submit a description of which review is yours in the app store. We will take
+    a look at it and the submission time in the store.
+
+### Share Zulip on alternativeto.net
+
+1. Get your parent’s permission to make an account, and fill out your profile.
+Link to at least one external social media account.
+
+2. Think of 2-4 products you use, and upvote (“Like”) them. To get to a product,
+search for it in the “I want to replace” bar in the upper right. Zulip's profile
+is [here](https://alternativeto.net/software/zulip-chat-server/) 
+
+3. For each of the products in 2, consider their competitors, and go to those
+competitors pages and list those products as good alternatives. Zulip is
+considered by many to be a good alternative to
+[Slack](https://alternativeto.net/software/slack/)
+
+3. Send us a link to your profile (e.g.
```

Comment:
For this and the preceding analogous lines: I think `us` should be more specific, e.g. put all the links and review pointers  in some text file and submit that file on the GCI app.